game/player.py

```diff
@@ -1,20 +1,23 @@
 #!/usr/bin/env python
-from abc import ABCMeta, abstractmethod
-from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
 
 class Player(object):
-    __metaclass__ = ABCMeta
 
-    def __init__(self, color):
+    def __init__(self, color, strategy=None):
+        if strategy:
+            self._strategy = strategy
+        else:
+            self._strategy = StrategyOne()
         self._color = color.lower()
 
     def color(self):
         return self._color
 
+    def set_strategy(self, new_strategy):
```

Comment:
You're right, we already discussed this during our last meeting. I just forgot to remove it :P